LoopKitUI/Extensions/Color.swift

```diff
@@ -8,13 +8,35 @@
 
 import SwiftUI
 
+// MARK: - Color palette for common elements
+/// The intent in providing these suggested context specific colors is to inform the user of the related context of the UI. Note loopAccent is intended to be use as the app accent colour.
 extension Color {
-    public static let destructive = red
+    public static let carbs = Color(UIColor.carbs)
+    
+    public static let critical = Color(UIColor.critical)
 
-    public static let severeWarning = red
-    public static let warning = Color(UIColor(dynamicProvider: { traitCollection in
-        return traitCollection.userInterfaceStyle == .dark
-            ? UIColor(red: 0.902, green: 0.494, blue: 0.039, alpha: 1)
-            : UIColor(red: 0.863, green: 0.455, blue: 0, alpha: 1)
-    }))
+    public static let glucose = Color(UIColor.glucose)
+
+    public static let insulin = Color(UIColor.insulin)
+    
+    public static let loopAccent = Color(UIColor.loopAccent)
+
+    public static let warning = Color(UIColor.warning)
+}
+
+// MARK: - Context for colors
+extension Color {
+    public static let cobTintColor = carbs
+
+    public static let delete = critical
+    
+    public static let destructive = critical
```

Comment:
What's the semantic difference between `delete` and `destructive`?

Reply:
Good point! I was just keeping the existing terms, but I'll reduce these to just `destructive`.